openstates/metadata/README.md.sample

```diff
@@ -0,0 +1,35 @@
+# openstatesmetadata
+
+The `openstatesmetadata` package provides a lightweight solution for accessing legislative metadata from the OpenStates project without needing to install the entire core OpenStates project. It is designed for external projects that rely on this specific subset of data.
```

Comment:
`openstates-metadata` nit - should match the string from `name` property in `pyproject.toml.sample` - also several lines below need find/replace

Reply:
Hmm, I did change the name to `openstatesmetadata` because `openstates-metadata` somehow fails to upload. 
I just tested again now with `openstates_metadata` and that works. it looks like hyphen is not accepted in a package name